index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&family=Roboto:ital,wght@0,100;0,300;0,700;1,300;1,500&display=swap" rel="stylesheet">
+  <!-- Material Icons -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+  
+<div class = main_section>
+ 
+<header class="nav-bar">    
+  <div class="logo-left">My Logo</div>
+ <nav><i class="material-icons">menu</i></nav>
+</header>
+
+<section class="headline">
+  <div class="container">
+   <h1>I'm Cookey<br>Glad to see <br> you!</h1>
+    <p>I’m a software developer! I can help you build a product , feature or website look through some of my work and experience!  If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+
+  <h5>LET'S CONNECT</h5>
+
+  <div class="contacts">
+   
+    <a href="">
+      <img src="./images/twitter.svg" class="social-icons" alt="twitter">
+    </a>
+    <a href="">
+      <img src="./images/linkedin.svg" class="social-icons" alt="twitter">
+    </a>
+    <a href="">
+      <img src="./images/medium.svg" class="social-icons" alt="twitter">
+    </a>
+    <a href="">
+      <img src="./images/github.svg" class="social-icons" alt="twitter">
+    </a>
+    <a href="">
+      <img src="./images/Vector.svg" class="social-icons" alt="twitter">
+    </a>
+  </div>
```

Comment:
- Kindly enclose this block with a `ul` tag and each social icon should have an `li` tag as requested by the previous reviewer 🏄